### ISSUE TITLE
Middle click always turns on auto

### ISSUE THREAD

**Issue by mxmilkiib** [Fushko/gammy#87]
I'd like to save on performance and energy by having auto off, but middle clicking to toggle the state toggles between auto being on and off, rather than the current settings vs off. Could this be possible?

**Comment by Fushko** [comment]
Yes, ideally if you have e.g. brightness at 70%, toggling on and off should revert to 70% instead of 100%. Same with temperature. This is what you meant, right?

**Comment by mxmilkiib** [comment]
I'd like to quite frequently switch between off and non-auto especially when I'm watching video, but sometimes use auto.

Maybe an option for the user so they could choose what toggle action happens on middle click?
